src/components-Todo/ErrorNotification.tsx

```diff
@@ -0,0 +1,56 @@
+import classNames from 'classnames';
+import { useEffect } from 'react';
+
+/* eslint-disable jsx-a11y/control-has-associated-label */
+interface Props {
+  hasLoadError: boolean;
+  setHasLoadError: (event: boolean) => void;
+}
+
+export const ErrorNotification: React.FC<Props> = ({
```

Comment:
create enum for  errors texts
Pass the error text in props for `ErrorNotification` and use it instead of instant text)